addons/settings/fnc_readFile.sqf

```diff
@@ -0,0 +1,32 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_settings_fnc_readFile
+
+Description:
+    Reads and applies settings from a file.
+    Only considered as public API from a static settings addon!
```

Comment:
Should stay an internal function. Loading the userconfig file could be some kind of ingame setting itself.